pkgs/development/python-modules/shot-scraper/default.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, click
+, click-default-group
+, pytestCheckHook
+, hypothesis
+, pytest-mock
+, playwright
+, pyyaml
+}:
+
+buildPythonPackage rec {
```

According to the documentation:

> A command-line utility for taking automated screenshots of websites

As it's not a library, it should be handled as a tool. 

```suggestion
python3.pkgs.buildPythonApplication rec {
```

Perhaps move it to `tools/graphics/`

---

Thanks @fabaff, I've moved this to the other directory :)